generic/piranha/src/config.rs

```diff
@@ -52,20 +52,20 @@ pub(crate) fn read_config_files(
   // Read the language specific cleanup rules and edges
   let language_rules: Rules = read_toml(
     &path_to_language_specific_cleanup_config.join("rules.toml"),
-    false,
+    true,
```

Comment:
This allows cleanup rules for a language to be empty.  Similar for the below three similar updates.